package.json

```diff
@@ -108,5 +108,8 @@
     "fail": [
       "@semantic-release/github"
     ]
+  },
+  "resolutions": {
+    "react-helmet/react-side-effect": "2.1.0"
```

Comment:
it's only in this library the problem?

Reply:
yes, I've searched through and found only helmet using it.